internal/bus/message_pipe.go

```diff
@@ -46,20 +46,20 @@ type (
 		bus            messagebus.MessageBus
 		messageChannel chan *Message
 		plugins        []Plugin
-		mu             sync.RWMutex
+		pluginsMutex   sync.Mutex
 	}
 )
 
 func NewMessagePipe(size int) *MessagePipe {
 	return &MessagePipe{
 		messageChannel: make(chan *Message, size),
-		mu:             sync.RWMutex{},
+		pluginsMutex:   sync.Mutex{},
```

Comment:
Why the switch to a `Mutex` instead of a `RWMutex`?

Reply:
switched to be consistent with other plugins